src/bindings/python/src/openvino/preprocess/torchvision/requirements.txt

```diff
@@ -0,0 +1,4 @@
+-c ../../../../constraints.txt
+torch
+torchvision
+pillow
```

Comment:
is it supposed to be installed with the product?
if yes, how users will receive these constraints?
if no, we need to add this file to exclude for installation rules.

Reply:
That's a good question. The idea is that we do not force the user to install these packages for runtime usage, as they are very big. Instead we warn the user that these packages are required in case user wants to use the converter: 
https://github.com/openvinotoolkit/openvino/pull/17934/files#diff-86e57e225a38a87f0702fd01cc79f91965287836e863844e7037b548b3dd9a9cR42

Because of that, at least before we introduce different build system, I'd actually opt for removing the version constraints as you mentioned.

> if no, we need to add this file to exclude for installation rules.

Could you please point me to a file where I can do that?

Reply:
> Could you please point me to a file where I can do that?

In cmake files of python project you need to run an `install` rule for this directory and add exclude filter.
Maybe we just to change the location of this file not to be in the directory copied to a product?

Reply:
The file has been excluded from installation.